modular_skyrat/modules/contractor/code/items/tablet.dm

```diff
@@ -1,17 +1,21 @@
+/obj/item/modular_computer/tablet/syndicate_contract_uplink/preset/uplink
+	starting_programs = list(
+		/datum/computer_file/program/contract_uplink,
+		/datum/computer_file/program/crew_manifest,
+	)
+	device_theme = "syndicate"
+
+	// Contractor uplink system board has the user's data baked directly into it on creation
+	var/datum/opposing_force/opfor_data
```

Comment:
null this in Destroy()

Reply:
Fixed. Occurs to me this most likely leaked before.